src/main/java/com/conveyal/datatools/editor/controllers/api/EditorController.java

```diff
@@ -134,9 +138,10 @@ private String deleteMultipleTrips(Request req, Response res) {
             // Commit the transaction after iterating over trip IDs (because the deletes where made without autocommit).
             tableWriter.commit();
             LOG.info("Deleted {} trips", tripIds.length);
-        } catch (Exception e) {
-            e.printStackTrace();
-            haltWithMessage(req, 400, "Error deleting entity", e);
+        } catch (InvalidNamespaceException e) {
+            haltWithMessage(req, 400, "Invalid namespace");
+        } catch (SQLException e) {
```

Comment:
I understand why the exception subtypes are being explicitly listed, but can we guarantee that a NPE won't be thrown in this try block? I worry that exceptions will happen that are not caught and will just fail silently. Can you reassure me that won't happen? Otherwise I think it's better to just catch the generic `Exception`.

Reply:
Yeah, probably a good idea to do that here if the exact failure reason isn't obvious.